opa-audit-operator/metadata.yaml

```diff
@@ -0,0 +1,6 @@
+name: gatekeeper-audit
+description: |
+  This charm deploys opa.
+summary: |
+  Kubernetes opa operator
```

Comment:
Description and summary need to be updated. This charm deploys Gatekeeper Audit.

Reply:
updated